synnax/pkg/api/api.go

```diff
@@ -210,7 +210,7 @@ type API struct {
 	config       Config
 	Auth         *AuthService
 	User         *UserService
-	Telem        *FrameService
+	Framer       *FrameService
```

Comment:
@emilbon99 Will changing the name to `Framer` from `Telem` cause any problems with incompatibility?